<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuralNetworkDemoWindow</class>
 <widget class="QMainWindow" name="NeuralNetworkDemoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="_tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Sine Curve</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="GraphWidget" name="_sinePlotWidget" native="true"/>
        </item>
        <item>
         <widget class="GraphWidget" name="_sineNetArchitectureWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>MNIST</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="_mnistResultsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Epochs:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_trainEpochsBox">
        <property name="maximum">
         <number>100000000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Batch Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="_trainBatchSizeBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>5000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="_trainProgressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v of %m</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_trainSineNetButton">
        <property name="text">
         <string>Train</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>GraphWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
